tasks/ArtifactoryMaven/Ver2/mavenBuild.js

```diff
@@ -72,7 +72,7 @@ function createMavenConfigFile(cliPath, buildDir) {
     // Configure resolver server, throws on failure.
     let artifactoryResolver = tl.getInput('artifactoryResolverService');
     let cliCommand = utils.cliJoin(cliPath, mavenConfigCommand);
-    if (artifactoryResolver != null) {
+    if (!!artifactoryResolver) {
```

Comment:
I think both expressions are equal when they are inside `if`. The not-not operator converts expressions to booleans. `if` already knows how to handle truthy and falsy values. 
```suggestion
    if (artifactoryResolver) {
```